pkgs/applications/kde/kdepim-runtime/default.nix

```diff
@@ -23,4 +23,7 @@ mkDerivation {
     pimcommon libkgapi libsecret
     qca-qt5 qtkeychain qtnetworkauth qtspeech qtxmlpatterns
   ];
+  qtWrapperArgs = [
+    "--prefix SASL_PATH : ${lib.makeSearchPath "lib/sasl2" [ cyrus_sasl libkgapi ]}"
```

Comment:
you probably want `cyrus_sasl.out` here, `cyrus_sasl` uses the bin output (there's no lib/sasl2 there), which makes all plugins unavailable.

Reply:
Yeah, you are right. Will fix it now.

Reply:
#190502